pages/AboutSolicitorPage.js

```diff
@@ -0,0 +1,27 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    solicitorsName: '#PetitionerSolicitorName',
+    firmName: '#PetitionerSolicitorFirm',
+    firmDxAddress: '#DerivedPetitionerSolicitorAddr',
+    yourReferenceNumber: '#D8SolicitorReference',
+    phoneNumber: '#PetitionerSolicitorPhone',
+    email: '#PetitionerSolicitorEmail',
+    solicitorAgree: '#SolicitorAgreeToReceiveEmails-Yes',
+    submit: 'button[type="submit"]',
+  },
+
+  fillFormAndSubmit() {
+    I.fillField(this.fields.solicitorsName, 'PAPA AJASCO');
+    I.fillField(this.fields.firmName, 'ELEMU GETON');
+    I.fillField(this.fields.firmDxAddress, '100 Reede Road, RM10 8DU');
+    I.fillField(this.fields.yourReferenceNumber, 'AWS11234');
+    I.fillField(this.fields.phoneNumber, '07712345679');
+    I.fillField(this.fields.email, 'ccdsolicitorcase@pettyfrance.com');
+    I.click(this.fields.solicitorAgree);
+    I.wait(5)
```

Comment:
We have quite a lot of `wait` statements. Is `waitForAction` in codeceptjs config not enough?

Reply:
waitForAction is not enough. But I'll look into the wait() and try to reduce them for subsequents tests